.github/workflows/ci.yml

```diff
@@ -61,145 +68,58 @@ jobs:
-          # Output changes only depending on the git sha of the submodules
-          # hash of package/package.json for cache busting on release builds (version bump)
-          key: ${{ matrix.os }}-${{ hashFiles('submodules-status') }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}-${{ hashFiles('package/package.json') }}
-
-      - name: Setup Unity
-        uses: getsentry/setup-unity@46c2e082d98cc3a825a5b59038cb31705fe9ff56
-        id: setup-unity
-        with:
-          unity-version: ${{ matrix.unity-version }}
-          unity-version-changeset: ${{ matrix.unity-version-changeset }}
-
-      - name: Setup Unity Modules - Windows IL2CPP
-        if: matrix.os == 'windows-latest'
-        uses: getsentry/setup-unity@46c2e082d98cc3a825a5b59038cb31705fe9ff56
-        with:
-          unity-version: ${{ matrix.unity-version }}
-          unity-modules: windows-il2cpp
-
-      - name: Setup Unity Modules - macOS IL2CPP
-        if: matrix.os == 'macos-latest'
-        uses: getsentry/setup-unity@46c2e082d98cc3a825a5b59038cb31705fe9ff56
+      - uses: actions/download-artifact@v2
         with:
-          unity-version: ${{ matrix.unity-version }}
-          unity-modules: mac-il2cpp
-
-      - name: Setup Unity Modules - Android
-        uses: getsentry/setup-unity@46c2e082d98cc3a825a5b59038cb31705fe9ff56
+          name: Android-sdk
+          path: package-dev/Plugins/Android
+      - uses: actions/download-artifact@v2
         with:
-          unity-version: ${{ matrix.unity-version }}
-          unity-modules: android
-
-      - name: Setup Unity Modules - iOS
-        uses: getsentry/setup-unity@46c2e082d98cc3a825a5b59038cb31705fe9ff56
+          name: iOS-sdk
+          path: package-dev/Plugins/iOS
+      - uses: actions/download-artifact@v2
         with:
-          unity-version: ${{ matrix.unity-version }}
-          unity-modules: ios
+          name: Windows-sdk
+          path: package-dev/Plugins/Windows
 
       - name: Create Unity license config
-        shell: pwsh
         run: |
-          New-Item -Path '${{ matrix.unity-config-path }}' -ItemType Directory
-          Set-Content -Path '${{ matrix.unity-config-path }}services-config.json' -Value '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
-
-      # Need to write to $GITHUB_PATH to make the environment variable available to other steps.
-      - name: Add Unity on PATH
-        shell: pwsh
-        run: |
-          Write-Output '${{ matrix.unity-root }}${{ matrix.unity-version }}/${{ matrix.unity-path }}' | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-
-      # Step used to reduce the Unity path to avoid long path errors on Windows.
-      - name: Make symbolic link for Unity (Windows)
-        if: matrix.os == 'windows-latest'
-        shell: pwsh
-        run: |
-          New-Item -ItemType SymbolicLink -Path "C:\${{ matrix.unity-version }}" -Target "C:\Program Files\Unity\Hub\Editor\${{ matrix.unity-version }}"
```

Comment:
The reduced path is no longer required for Windows?

Reply:
it's not going to run on windows at the moment (in this "job") so this step would never run.

Reply:
resolving because the change was rolled back for now